### PR TITLE
[SDC-9767] Bugfix for avro generation

### PR DESCRIPTION
**Pull request by zlosim**
fixed avro generation issue and test added
https://issues.streamsets.com/browse/SDC-9767

### CODE REVIEW

**Comment by zlosim** [comment]
any thoughts on this one? @jarcec

File: commonlib/src/test/java/com/streamsets/pipeline/lib/generator/avro/TestAvroDataGenerator.java [hunk 1]
```diff
@@ -125,6 +126,8 @@
...
...\", \"fields\": [{ \"name\": \"name\", \"type\": [\"null\", \"string\"], \"default\": null }, { \"name\": \"parts\", \"type\": [\"null\", { \"type\": \"record\", \"name\": \"parts\", \"namespace\": \"\", \"fields\": [{ \"name\": \"required\", \"type\": [\"null\", { \"type\": \"record\", \"name\": \"parts\", \"namespace\": \"required\", \"fields\": [{ \"name\": \"name\", \"type\": [\"null\", \"string\"], \"default\": null }, { \"name\": \"params\", \"type\": [\"null\", { \"type\": \"record\", \"name\": \"parts\", \"namespace\": \"params.required\", \"fields\": [{ \"name\": \"size\", \"type\": [\"null\", \"string\"], \"default\": null }] }], \"default\": null }] }], \"default\": null }, { \"name\": \"optional\", \"type\": [\"null\", { \"type\": \"record\", \"name\": \"parts\", \"namespace\": \"optional\", \"fields\": [{ \"name\": \"name\", \"type\": [\"null\", \"string\"], \"default\": null }, { \"name\": \"params\", \"type\": [\"null\", { \"type\": \"record\", \"name\": \"parts\", \"namespace\": \"params.optional\", \"fields\": [{ \"name\": \"color\", \"type\": [\"null\", \"string\"], \"default\": null }] }], \"default\": null }] }], \"default\": null }] }], \"default\": null }] }";
```
Comment (jarcec) [inline commonlib/src/test/java/com/streamsets/pipeline/lib/generator/avro/TestAvroDataGenerator.java]: Do you mind breaking this schema into multiple lines for a better readability like we have for other schemas in this file?

Comment (zlosim) [reply commonlib/src/test/java/com/streamsets/pipeline/lib/generator/avro/TestAvroDataGenerator.java]: sure, changes are pushed

**Comment by jarcec** [comment]
Thank you for your contribution @zlosim !

**Pull request closed without merging**